app/controllers/episodes_controller.rb

```diff
@@ -28,6 +33,18 @@ def update
     end
   end
 
+  def add_start
+    @episode = Episode.find(params[:episode_id])
+    @episode.update(actual_start: Time.now)
+    redirect_to episode_path(@episode)
+  end
+
+  def add_finish
+    @episode = Episode.find(params[:episode_id])
```

Comment:
`set_episode` ? =)

Reply:
It takes :id as param, and route for this methods (add_start/add_finish) are built with :episode_id
So set_episode crashes for this method :(

Reply:
so solve it with some changes on route.rb